File: Bridge/Severylova-03/Bridge/Bridge/Circle.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Bridge
{
    class Circle : Shape
    {
        const double PI = 3.14;
        public double radius;

        public Circle(double _radius, Color color) : base(color)
        {
            if (_radius < 0)
            {
                throw new Exception("Radius must be positive");
            }
            this.radius = _radius;
        }

        public override double Perimeter()
        {
            return 2 * PI * radius;
        }

        public override double Area()
        {
            return PI * Math.Pow(radius, 2);
        }
    }
}

File: Bridge/Mozil/shapes/Circle.java
package shapes;

import colors.Color;

public class Circle extends Shape {
    private final double PI = 3.14;
    private double r;

    public Circle(Color color, double r) {
        super(color);
        this.r = r;

    }

    @Override
    protected double countArea() {
        return Math.pow(r, 2) * PI;
    }

    @Override
    protected double countPerimeter() {
        return 2 * PI * r;
    }
}
